tests/test_create_cross_validation_datalist.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import unittest
+from pathlib import Path
+
+import nibabel as nib
+import numpy as np
+
+from monai.data import create_cross_validation_datalist, load_decathlon_datalist
+
+
+class TestCreateCrossValidationDatalist(unittest.TestCase):
+    def test_content(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            datalist = []
+            test_image = nib.Nifti1Image(np.random.randint(0, 2, size=[128, 128, 128]), np.eye(4))
```

Comment:
don't think nibabel is really needed for this test, could you please simplify the tests?

Reply:
Sure, removed nibabel dependency in the latest commit.

Thanks.